src/components/Page.js

```diff
@@ -2,12 +2,24 @@
 import React, { useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
 
 import useRouter from '../utils/useRouter';
 
+const useStyles = makeStyles((theme) => ({
+  pageInner: {
```

Comment:
I am not sure if this style is creating the issue of the scroll bar or not. I tried to figure out how to solve it but wasn't able.
This scroll bar is appearing on all pages for me.

![image](https://user-images.githubusercontent.com/5901730/79106339-98b58c80-7d72-11ea-87de-aeac48bfc1de.png)

Reply:
Cool, I know the issue, good spot